app/(transparent_navbar)/_component/RetroSection.tsx

```
@@ -87,6 +113,24 @@ export default function RetroSection({
+                title={item.name}
+                category={item.category}
+                description={item.description}
+                teamSize={item.team.length}
+                opRecieve={item.reward ?? 0}
+                rank={item.ranking}
+                round="RetroPGF 4"
+                // vote={0}
+                icon={item.projectAvatarUrl}
+                banner={item.proejctCoverImageUrl}
+                isEligible={item.isEligibleFinal}
+                />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)